monai/handlers/lr_schedule_handler.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2020 MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from ignite.engine import Events
+
+
+class LrScheduleHander:
```

Comment:
LrScheduleHander -> LrScheduleHandler

Reply:
updated in the latest patch.
Thanks for correcting my typo.